modules/custom/d_commerce/README.md

```diff
@@ -28,4 +28,4 @@ provided by the `d_commerce_products_list` module.
 
 ## 3.1.0
 The way titles are displayed has been changed with the removal of the DrooplerProductVariation class,
-information on how to restore it can be found in web/UPDATE.md
+information on how to restore it can be found in [UPDATE.md](../../../../../../UPDATE.md)
```

Comment:
One level too far, please try to click the link on https://github.com/droptica/droopler/blob/589ac7837592cb4af2892755d6c566f175fbc457/modules/custom/d_commerce/README.md

Reply:
Please see the comment on `droopler_project` repo about it